docs/routing.md

```diff
@@ -26,56 +37,34 @@ This pattern is the same for all Calypso sections. If you're unfamiliar with `pa
 
 Colon-prefixed route parts (such as `:site_id` here) are called route parameters. More on them in a bit.
 
-## Middleware
+## Middlewares
```

Comment:
I've been told the plural here is a bit unidiomatic (cf. software :arrow_right: ~softwares~), so let's stick with the singular here.